### ISSUE TITLE
No longer compiles

### ISSUE THREAD

**Issue by Fancy2209** [saharan/OimoPhysics#67]
Cullprit is this function
```hx
static function filterFuncExpr(e:Expr):Expr {
		log("filterFuncExpr " + e);
		switch(e.expr) {
		case EVars(vars):
			log("EVars(" + vars + ")");
			var newVars:Array<Var> = vars.copy();
			for (v in vars) {
				log("v " + v);
				var names:Array<String> = v.name.names(v.type);
				log("names " + names);
				if (names != null) {
					newVars = newVars.concat(names.map(function(name) {
						return {
							name: name,
							type: macro:Float,
							expr: null,
							meta: null,
							isFinal: false
						};
					}));
				}
			}
			e.expr = EVars(newVars);
			return e;
		case _:
			log("others");
			return ExprTools.map(e, filterFuncExpr);
		}
	}
```
src/oimo/m/B.hx:182: lines 182-190 : error: { type : haxe.macro.ComplexType, name : String, meta : Null<haxe.macro.Metadata>, isFinal : Bool, expr : Null<haxe.macro.Expr> } has no field isStatic
src/oimo/m/B.hx:182: lines 182-190 : ... have: Array<{ type, name, meta, isFinal, expr }>
src/oimo/m/B.hx:182: lines 182-190 : ... want: Array<haxe.macro.Var>
src/oimo/dynamics/rigidbody/RigidBodyType.hx:6: characters 1-8 : Build failure

**Comment by mrwedders** [comment]
I've just made a [pull request](https://github.com/saharan/OimoPhysics/pull/69), but some new optional properties were introduced with Haxe 4.3 which just need to be added.

**Comment by Fancy2209** [comment]
Oh, thanks!

**Comment by saharan** [comment]
fixed in #69 !
